### PR TITLE
Introduce tablets option

### PR DESCRIPTION
**Pull request by dy**
Introduces `options` to detect tablets as 

```js
isMobile({ tablets: true })
isMobile(ua, {tablets: true})
```

This covers https://github.com/juliangruber/is-mobile/pull/4 and closes https://github.com/juliangruber/is-mobile/issues/3.

Tablets regexp is taken from http://detectmobilebrowsers.com/about.

Unlocks https://github.com/plotly/plotly.js/issues/280.

@juliangruber

### CODE REVIEW

**Comment by juliangruber** [comment]
- [ ] can you please fix the indentation (2 spaces)
- [ ] can you also please remove the comments? the code is still small enough
- [ ] `tablets` should be `tablet`, as `tablet` is a concept and not an amount

Otherwise, lgtm!

**Comment by dy** [comment]
@juliangruber thanks for the review, done!

File: index.js [hunk 1]
```diff
@@ -1,13 +1,29 @@
...
+
+function isMobile (ua, opt) {
+  if (ua && typeof ua != 'string') {
```
Comment (juliangruber) [inline index.js]: The top of the function was working well, please undo all changes until where the tablet option is checked. A PR is always most likely to merge if the changes are as minimal as possible.

If you still think that the top of the function should be refactored, please submit a separate PR for it 🙏

Comment (dy) [reply index.js]: @juliangruber sure, done

**Comment by juliangruber** [comment]
Thanks! Published as 1.1.0
